### DISCUSSION TITLE
Recordings are playing at fast rate / clipping

### DISCUSSION THREAD

**Discussion by israelgoldstein** [rtl-airband/RTLSDR-Airband#375] [Q&A]
I recently added more channels to my configuration file,

I also changed the FFT to 1024 from 512
I set the bandwidth to 5000
And I set the sample rate to 2.4

Recently some of my recordings when I am playing them, sounds like it's at a fast speed, or it's clipping, 

What would cause an issue like this ?

**Comment by charlie-foxtrot** [comment]
How many channels do you have running?

If you reduce the number of channels does the problem stop?

Can you upload an example somewhere?

If you look at the audio waveform clipping and “fast” are going to look different. Clipping is when the magnitude of the waveform exceeds max/min where as fast would be, well, fast :)

From your description I suspect you are loosing audio samples because you are driving the output thread faster than it can process.

Try turning on the stats file and see if you have buffer overruns - https://github.com/charlie-foxtrot/RTLSDR-Airband/wiki/Channel-usage-statistics

If you do have overruns, try enabling multiple output threads - https://github.com/charlie-foxtrot/RTLSDR-Airband/wiki/Multithreaded-operation

**Reply by israelgoldstein** [reply]
I restarted the application and the issue went away 

I'm running approximately 35 channels
